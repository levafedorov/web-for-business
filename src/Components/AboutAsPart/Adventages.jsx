import React from 'react';
import AdventageItem from './AdventageItem';

const adventageNames = ["friendly team", "involved", "assistance"];



export default function Adventages(props){
     
    

    return <div className="adventages">
        {adventageNames.map((item, i) => <AdventageItem name={item} key={`key-${i}`}/>)}
   </div>
}