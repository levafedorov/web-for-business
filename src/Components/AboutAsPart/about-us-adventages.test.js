import React from 'react';
import {shallow} from 'enzyme';
import Adventages from './Adventages';

describe("Adventages component", ()=>{
    test("Should component render", ()=>{
       const component = shallow(<Adventages/>);
       expect(component).toMatchSnapshot();    
    });
});

