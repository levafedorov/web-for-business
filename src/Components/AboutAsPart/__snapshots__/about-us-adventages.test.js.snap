// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adventages component Should component render 1`] = `
<div
  className="adventages"
>
  <AdventageItem
    key="key-0"
    name="friendly team"
  />
  <AdventageItem
    key="key-1"
    name="involved"
  />
  <AdventageItem
    key="key-2"
    name="assistance"
  />
</div>
`;
